app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,13 @@
+ <div class="">
+   <h1 class="heading">Recipes</h1>
+    <div class="" >
+      <%= link_to new_recipe_path do %>
+      <button class="add-btn">
+        New recipe
```

Comment:
- [ ] Following the given wireframe for the public page, let's ensure that the name of the creature of a given recipe is displayed on the public page accordingly. 

**The wireframe public page:**
![image](https://user-images.githubusercontent.com/12745474/218155185-570dffd5-1594-4ef3-a43a-7855c836afb9.png)